src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -355,7 +357,13 @@ export default React.memo(function GamePage(): JSX.Element | null {
                     <DotsMenu gameInfo={gameInfo} handleUpdate={handleUpdate} />
                   </div>
                   <div className="infoWrapper">
+                    <Genres genres={wikiInfo?.pcgamingwiki?.genres} />
```

Comment:
```suggestion
                    <Genres genres={wikiInfo?.pcgamingwiki?.genres || []} />
```

just a suggestion so we don't have to consider `undefined` for this https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3330/files#diff-b822c7f041f98de8200abf8822131fecf49d7e6fa1456370c34a738d604bc76eR8, makes it a bit simpler

Reply:
true, will fix